documentation/modules/ROOT/pages/operations/debezium-platform.adoc

```diff
@@ -508,46 +553,63 @@ The pipeline section is the place where you connect the "dots". You can define w
 [.responsive]
 video::cY7TS0nWBoY[youtube, title="Create, edit and remove a pipeline"]
 
-==== Creating pipelines
+==== Creating pipeline
 
-. From the platform UI, open the **Pipeline** menu and click **Create your first pipeline**  to open the **Pipeline Designer**.
-+
-From the **Pipeline Designer** you can add the pieces for your data pipeline.
-. Click the **+ Source** box to add a source, and then choose a previously created source, or create a new source.
+From the platform UI, open the **Pipeline** menu and click **Create your first pipeline** to open the **Pipeline Designer**.
+From the Pipeline Designer you can add the pieces for your data pipeline.
 
-Click the **+ Destination** box to add a destination .
+. Click the **+ Source** box to add a source, and then choose a previously created source, or create a new source.
```

Comment:
The double asterisks do work, but a pair of single asterisks is sufficient to apply bold formatting.